source/material_model/averaging.cc

```diff
@@ -321,17 +321,14 @@ namespace aspect
        */
       if (out.viscosities.size() > 1)
```

Comment:
@bangerth can you explain this logic to me? The only constructor of ```MaterialModelOutputs``` gets the number of quadrature points, so checking for 1 doesn't make sense to me.